README.md

```diff
@@ -60,7 +62,7 @@ fn zero_to_three() -> impl Stream<Item = u32> {
 #[tokio::main]
 async fn main() {
     let s = zero_to_three();
-    pin_mut!(s); // needed for iteration
+    pin_mut!(s); // Pinning is needed for iteration
```

Comment:
Since, `impl Stream` above doesn't retrun a `impl Stream + Unpin` we must ensure that the stream is `Unpin` to await something on it.

Might be good to explain that.